Formula/spidermonkey.rb

```diff
@@ -20,49 +21,52 @@ class Spidermonkey < Formula
     sha256 cellar: :any, mojave:   "8c0b46bc04a7e95f99262969b22cc311ee1f7d83413af05865318743ccd96944"
   end
 
-  depends_on :macos # Due to Python 2
+  depends_on "autoconf@2.13" => :build
```

Comment:
Why doesn't this work with `autoconf` or `autoconf@2.69`?

Reply:
Mozilla project requires autoconf 2.13 https://bugzilla.mozilla.org/show_bug.cgi?id=297544

Reply:
Ah, yes. `gjs` also depends on autoconf 2.13.